src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1834,7 +1838,40 @@ public override BoundNode VisitConditionalOperator(BoundConditionalOperator node
                 }
             }
 
-            _resultType = TypeSymbolWithAnnotations.Create(resultType ?? node.Type.SetUnknownNullabilityForReferenceTypes(), isNullableIfReferenceType);
+            var resultTypeWithAnnotations = TypeSymbolWithAnnotations.Create(resultType ?? node.Type.SetUnknownNullabilityForReferenceTypes(), isNullableIfReferenceType);
+
+            if ((object)resultType != null)
+            {
+                if (!isConstantFalse)
+                {
+                    ApplyConversion(
+                    node.Consequence,
+                    consequence,
+                    consequenceConversion,
+                    resultTypeWithAnnotations,
+                    consequenceResult,
+                    checkConversion: true,
+                    fromExplicitCast: false,
+                    useLegacyWarnings: false,
+                    AssignmentKind.Assignment);
```

Comment:
Indent arguments.

Reply:
D'uh, sorry.
Btw, the new `ApplyConversion` and `VisitOptionalImplicitConversion` model is much better. It made this PR easier and I expect many more to come. Thanks!